package.json

```diff
@@ -389,13 +389,13 @@
       "simulator": {
         "type": "ios.simulator",
         "device": {
-          "type": "iPhone SE (2nd generation)"
+          "type": "iPhone SE (3rd generation)"
```

Comment:
ah, please don't commit this. the CI isn't ready for this change

Reply:
Didn't intend to commit this, I thought I've excluded it. Slipped, will fix.